themes/jquery/functions.php

```diff
@@ -251,3 +251,36 @@ function jq_image_posted_on() {
 
 	return $classes;
 } );
+
+/**
+ * Content Security Policy
+ */
+function jq_content_security_policy() {
+  $nonce = wp_create_nonce( JQUERY_LIVE_SITE );
+  $policy = array(
+    'default-src' => "'self'",
+    'script-src' => "'self' 'nonce-$nonce' code.jquery.com",
+    'style-src' => "'self' 'nonce-$nonce' 'sha256-biLFinpqYMtWHmXfkA1BPeCY0/fNt46SAZ+BBk5YUog=' 'unsafe-hashes'",
```

Comment:
This is the SHA for the inline style from typesense. `unsafe-hashes` is required in order to use this method in `style-src`. Still, it's better than allowing `unsafe-inline` everywhere.